src/config/env/dev.json

```diff
@@ -1,5 +1,6 @@
 {
-  "CHAIN_ID": "5",
+  "NETWORK": "sepolia",
```

Comment:
The network name can be inferred by the chainId. Can we avoid adding an extra variable?